.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Standard Ruby Linter
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
```

Comment:
this is run on push and pull request…i don’t know if we need both, but for a second there, i didn’t see it in the PR box down below, now i do see it.

Reply:
i think `pull_request` [without any specific type means it'll run the linter on `opened`, `synchronized`, and `reopened` by default ](https://github.com/orgs/community/discussions/25161#discussioncomment-7453899)

as for `push`, we may need it for `main` branch specifically, if we're not running this in the deploy job.